packages/catalog-client/src/CatalogClient.ts

```diff
@@ -177,7 +177,13 @@ export class CatalogClient implements CatalogApi {
   ): Promise<GetEntitiesByRefsResponse> {
     const response = await this.apiClient.getEntitiesByRefs(
       {
-        body: request,
+        body: {
+          entityRefs: request.entityRefs,
+          fields: request.fields,
+        },
+        query: {
```

Comment:
it's arguably a bit awkward to see this mixture of query and body, but it is what it is for now, until we've picked a good json encoding of the filter format

Reply:
Agree, but if it works and is light implementation wise, why not 🤷